build-logic/dependency-modules/src/main/kotlin/gradlebuild/modules/extension/ExternalModulesExtension.kt

```diff
@@ -251,6 +252,7 @@ abstract class ExternalModulesExtension {
         testng to License.Apache2,
         tomlj to License.Apache2,
         trove4j to License.LGPL21,
+        agp to License.Apache2,
```

Comment:
Do we need this? I thought we only need this for things we bundle in our distribution.